doc/source/testing.rst

```diff
@@ -143,18 +166,18 @@ the next gold standard iteration.  You can see the current iteration by looking
 in the ``.travis.yml`` file at the ``TRIDENT_GOLD`` entry--enumerate this and
 tag the changeset.  Update the ``.travis.yml`` file so that the ``YT_GOLD`` and
 ``TRIDENT_GOLD`` entries point to your desired changeset and tag.  You have to 
-explicitly push the new tag (hereafter ``test-standard-v2``) to 
+explicitly push the new tag (hereafter ``test-standard-v3``) to 
 the repository.
 
 .. code-block:: bash
 
-   $ git tag test-standard-v2 <trident-changeset>
+   $ git tag test-standard-v3 <trident-changeset>
    $ ... edit .travis.yml files to update YT_GOLD=<yt changeset>
-   $ ... and TRIDENT_GOLD=<test-standard-v2
+   $ ... and TRIDENT_GOLD=test-standard-v3
    $ git add .travis.yml
    $ git commit
    $ git push origin
-   $ git push origin test-standard-v2
+   $ git push origin test-standard-v3
```

Comment:
Perhaps this step should be in a separate set of instructions that occurs after the above steps have been included in a pull request that then has been merged.

Reply:
Good idea.  I tried to address this as you suggested.